ui/src/assets/i18n/en.json

```diff
@@ -136,7 +136,13 @@
           "tooltip": "When was the event communicated (local time for port)"
         },
         "eventDeliveryStatus": {
-          "tooltip": "Delivery status of the timestamp"
+          "label": "status of Timestamp",
+          "tooltip": {
+            "green": "Green: event is ingoing",
+            "yellow": "Yellow: event is pending for delivery",
+            "red": "Red: event is attempting delivery",
+            "grey": "Grey: event is outgoing"
```

Comment:
Lets use `timestamp` instead of `event`  (because the user expects `timestamp` in this content).  Also `Red` should probably be `Red: Error while latest attempt to deliver the timestamp (will retry later)`.

Reply:
@AndreasHoff You're welcome to disregard my comments and do as above.